packages/api/tests/endpoints/collections/list-collections.js

```diff
@@ -86,34 +115,23 @@ test('CUMULUS-912 GET without pathParameters and with an invalid access token re
 test.todo('CUMULUS-912 GET without pathParameters and with an unauthorized user returns an unauthorized response');
 
 test.serial('default returns list of collections from query', async (t) => {
-  const stub = sinon.stub(EsCollection.prototype, 'query').returns({ results: [t.context.testCollection] });
-  const spy = sinon.stub(EsCollection.prototype, 'addStatsToCollectionResults');
-
   const response = await request(app)
     .get('/collections')
     .set('Accept', 'application/json')
     .set('Authorization', `Bearer ${jwtAuthToken}`)
     .expect(200);
 
   const { results } = response.body;
-  t.is(results.length, 1);
-  t.is(results[0].name, t.context.testCollection.name);
-  t.true(spy.notCalled);
-  stub.restore();
-  spy.restore();
+  t.is(results.length, 10);
 });
 
 test.serial('returns list of collections with stats when requested', async (t) => {
```

Comment:
So based on what happens with includeStats, I'm thinking this may need to change a bit. If it needs ES, I might suggest creating a separate temporary test file, since all the setup is a bit different.

Reply:
well, that is the understanding I have from the other conversation with Jenny.